AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOperation.swift

```diff
@@ -54,7 +54,23 @@ final class InitialSyncOperation: AsynchronousOperation {
         }
 
         let lastSyncMetadata = getLastSyncMetadata()
-        lastSyncTime = lastSyncMetadata?.lastSync
+        guard let lastSync = lastSyncMetadata?.lastSync else {
+            return
+        }
+
+        //TODO: Update to use TimeInterval.milliseconds when it is pushed to master branch
```

Comment:
Link the todo to a backlog item

Reply:
Opened issue here:
https://github.com/aws-amplify/amplify-ios/issues/398

And updated code.